test/jdk/java/awt/Dialog/SiblingChildOrder/SiblingChildOrderTest.java

```diff
@@ -26,7 +26,7 @@
  * @bug 8190230 8196360
  * @summary [macosx] Order of overlapping of modal dialogs is wrong
  * @key headful
- * @run main SiblingChildOrderTest
+ * @run main/othervm -Dsun.java2d.uiScale=1 SiblingChildOrderTest
```

Comment:
Switching this tests to scale=1 seems completely wrong. On macos it just tweak the rendering so that bug became invisible due to lost of precision. And now we will run the tests in lowDPI on all other platforms.

Reply:
If really needed I can file a follow-on to change this test to problem listed on macOS like some of the others .. but this test doesn't look to me like its a pixel precision test ...